### ISSUE TITLE
Replace (find panel) ignoring case of replacement text if character matches other than case

### ISSUE THREAD

**Issue by rohrlich** [goki/gide#245]
Ignore should only apply to the find and not the replace.

Try this for example in ra25.go

- Bring up find dialog and enter AlphaCyc in Find field
- Set to Ignore Case
- Enter aLpHacyc in Replace field
- Click Ok
- Click "All" next to replace field
- Click Ok for the confirm dialog
- Notice that the instances of AlphaCyc have not changed to aLpHacyc

Likewise if you use AlphaCyc for find and alPha for replacement the result is Alpha

Now get this!!
I entered "Learning Phonemes" for find and "Sensitivity to Sequential Probabilities" for replace and got "SensitiviTy to sequential Probabilities". Notice that the replace has uppercased the 10th character 'T' while in the find it is the 10th character 'P' that is uppercase!!

**Comment by rcoreilly** [comment]
Yep it is using "MatchCase" when Ignore is used -- this works great most of the time, and is what emacs does for example.  Could maybe be made smarter though in terms of treating caps after space as a special case and matching that in the replacement.

**Comment by rohrlich** [comment]
Ok, well you can decide if you want to close this. Easy enough to uncheck ignore case for particular replace cases.
